sound/shairport-sync/Makefile

```diff
@@ -29,7 +30,7 @@ define Package/shairport-sync/default
   SECTION:=sound
   CATEGORY:=Sound
   TITLE:=AirPlay compatible audio player
-  DEPENDS:=@AUDIO_SUPPORT +libpthread +alsa-lib +libconfig +libdaemon +libpopt
+  DEPENDS:=@AUDIO_SUPPORT +libpthread +alsa-lib +libconfig +libdaemon +libpopt +libstdcpp +libplist +libsodium +libgcrypt +libffmpeg-full +libuuid +nqptp
```

Comment:
So many dependencies right now. Do you really need full variant of ffmpeg?

Reply:
Unfortunately, yes. I tested with both `libffmpeg-audio-dec` and `libffmpeg-mini`, but neither of those provide the required `libswresample.so`

Reply:
Okay, but this file is also provided by libffmpeg-custom. So, this makes it as hard dependency for full. 
What about other dependencies? Why they are required now?

Reply:
The newly added dependencies are because of the new AirPlay2 support. I went back and checked each one to be sure.

I'm new to packaging with OpenWrt. Can you recommend the best way to add a dependency on either libffmpeg-custom or libffmpeg-full?